inst/app.crunch.io/api/datasets/3.json

```diff
@@ -20,7 +20,8 @@
     },
     "fragments": {
         "table": "https://app.crunch.io/api/datasets/3/table/",
-        "preferences": "https://app.crunch.io/api/datasets/3/preferences/"
+        "preferences": "https://app.crunch.io/api/datasets/3/preferences/",
+        "pk": "https://app.crunch.io/api/datasets/3/pk/"
```

Comment:
What i mean is that inst/app.crunch.io/api/datasets/3/pk.json appears empty, but it should be `
{"element": "shoji:entity", "self": "https://app.crunch.io/api/datasets/3/pk/", "description": "Allows a user to get and set the primary key for a dataset", "body": {"pk": []}}`